etc/amazon-eks-vpc.yaml

```diff
@@ -0,0 +1,167 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Should this be changed to more recent - say '2018-10-23'

Reply:
No, that should be left as 2010-09-09. 2018-08-30 is the date it was updated/released.